cla_backend/apps/reports/sql/MIExtractByOutcomeExtended.sql

```diff
@@ -0,0 +1,173 @@
+  ,null as "Complaint_Type"
+  ,null as "Complaint_Date"
+  ,null as "Complaint_Owner"
+  ,null as "Complaint_Target"
+  ,null as "Complaint_Subject"
+  ,null as "Complaint_Classification"
+  ,null as "Complaint_Outcome"
+  ,pd.contact_for_research as "Agree_Feedback"
+  ,c.exempt_user_reason as "Exempt_Client"
+  ,CASE upper(adapt.language) WHEN upper('Welsh') THEN true ELSE false END as "Has_Language"
+  ,adapt.language as "Language"
+  ,log.created as "Outcome_Created_At"
+  ,u.username as "Username"
+  ,c.thirdparty_details_id::bool as "Has_Third_Party"
+  ,ceil(EXTRACT(EPOCH FROM operator_first_action.created-c.created)) as "Time_to_OS_Action"
+  ,cc_org.name as organisation
+  ,c.notes as "Notes"
+  ,c.provider_notes as "Provider Notes"
+  -- diversity fields --
+  ,{diversity_expression} as diversity_json
+from cla_eventlog_log as log
+  JOIN legalaid_case as c on c.id = log.case_id and not (c.eligibility_check_id is null and c.diagnosis_id is null and personal_details_id is null)
+  LEFT OUTER JOIN legalaid_personaldetails as pd on c.personal_details_id = pd.id
+  LEFT OUTER JOIN cla_provider_provider as provider on c.provider_id = provider.id
+  LEFT OUTER JOIN diagnosis_diagnosistraversal as diagnosis on c.diagnosis_id = diagnosis.id
+  LEFT OUTER JOIN legalaid_category as category on diagnosis.category_id = category.id
+  LEFT OUTER JOIN timer_timer as timer on log.timer_id = timer.id and timer.stopped IS NOT null and timer.cancelled = false
+  LEFT OUTER JOIN legalaid_mattertype as mt1 on mt1.id = c.matter_type1_id
+  LEFT OUTER JOIN legalaid_mattertype as mt2 on mt2.id = c.matter_type2_id
+  LEFT OUTER JOIN legalaid_eligibilitycheck as ec on c.eligibility_check_id = ec.id
+  LEFT OUTER JOIN legalaid_adaptationdetails as adapt on c.adaptation_details_id = adapt.id
+  LEFT OUTER JOIN auth_user as u on log.created_by_id = u.id
+  LEFT OUTER JOIN call_centre_operator as op on u.id = op.user_id
+  LEFT OUTER JOIN cla_provider_staff as staff on u.id = staff.user_id
+  LEFT OUTER JOIN legalaid_mediacode as mc on mc.id = c.media_code_id
+  LEFT OUTER JOIN latest_outcome on latest_outcome.case_id = c.id and latest_outcome.rn = 1
+  LEFT OUTER JOIN operator_first_action on operator_first_action.case_id = c.id and operator_first_action.rn = 1
+  LEFT OUTER JOIN operator_first_access on operator_first_access.case_id = c.id and operator_first_access.rn = 1
+  LEFT OUTER JOIN provider_first_view on provider_first_view.case_id = c.id and provider_first_view.rn = 1
+  LEFT OUTER JOIN provider_first_assign on provider_first_assign.case_id = c.id and provider_first_assign.rn = 1
+  LEFT OUTER JOIN legalaid_case split_case on c.from_case_id = split_case.id
+  LEFT OUTER JOIN cla_provider_provider as assigned_provider on (log.context->>'provider_id')::numeric = assigned_provider.id
+  LEFT OUTER JOIN call_centre_organisation AS cc_org ON cc_org.id = c.organisation_id
+where
+  log.type = 'outcome'
+  and log.created >= %s
+  and log.created < %s
```

Comment:
Just a quick query regarding this line, should this line not read `and log.created <= %s`

With the current implementation, entering the values:
Date from: 15/09/2023
Date to: 15/09/2023

Would always return no results, where the expected behaviour may be to generate a report of all today's cases.

Many thanks

Reply:
This is the current behaviour of the other report that this comes from so reluctant to change this.

Reply:
Sounds good, makes sense to match the current report implementation.